### DISCUSSION TITLE
This is realtek driver，the tl-wn726n is working

### DISCUSSION THREAD

**Discussion by xiyuxy** [chris1111/Wireless-USB-Big-Sur-Adapter#264] [Q&A] [answered]
Can you add this to your project?

http://www.comfast.com.cn/index.php?m=content&c=index&a=show&catid=30&id=595

**Comment by chris1111** [comment, marked as answer]
Its already support

**Reply by xiyuxy** [reply]
When I use RtlWlanU.kext it's not working.why?

**Comment by chris1111** [comment]
Use the program as it should and that's it
